playbook.yml

```diff
@@ -0,0 +1,34 @@
+---
+- hosts: all
+  gather_facts: False
+  become: True
+  pre_tasks:
+
+    # Check Ubuntu release version to determine if we need to install python 2,
+    # an Ansible dependency that isn't included by default in Ubuntu 16.04 and
+    # up.
+
+    - name: Check Ubuntu release
+      raw: cat /etc/lsb-release | grep DISTRIB_RELEASE | cut -d "=" -f2
+      register: ubuntu_release
```

Comment:
I needed to tag the `raw` tasks so that `ansible-lint` would avoid throwing false-positives:

```diff
diff --git a/playbook.yml b/playbook.yml
index e23f382..2cddcc0 100644
--- a/playbook.yml
+++ b/playbook.yml
@@ -11,6 +11,8 @@
     - name: Check Ubuntu release
       raw: cat /etc/lsb-release | grep DISTRIB_RELEASE | cut -d "=" -f2
       register: ubuntu_release
+      tags:
+        - skip_ansible_lint
 
     - debug: msg="Running Ubuntu version {{ ubuntu_release.stdout|float }}"
 
@@ -20,6 +22,8 @@
     - name: Update APT cache
       raw: apt-get update
       become: True
+      tags:
+        - skip_ansible_lint
 
     - name: Install python
       raw: apt-get install -yq python
@@ -31,4 +35,4 @@
```

Example output without these tags applied:

```
Idempotence test passed.
--> Executing ansible-lint.
[ANSIBLE0012] Commands should not change things if nothing needs doing
playbook.yml:11
Task/Handler: Check Ubuntu release

[ANSIBLE0012] Commands should not change things if nothing needs doing
playbook.yml:22
Task/Handler: Update APT cache
```